sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -13,20 +13,28 @@ namespace Azure.Communication
     /// </summary>
     public class CommunicationTokenRefreshOptions
     {
-        internal bool RefreshProactively { get; }
-        internal Func<CancellationToken, string> TokenRefresher { get; }
-
+        /// <summary>Default proactive refreshing interval in minutes.</summary>
+        public const int DefaultExpiringOffsetMinutes = 5;
         /// <summary>The initial token.</summary>
         public string InitialToken { get; set; }
-
-        private Func<CancellationToken, ValueTask<string>> _asyncTokenRefresher;
-
         /// <summary>The asynchronous token refresher.</summary>
         public Func<CancellationToken, ValueTask<string>> AsyncTokenRefresher
         {
             get => _asyncTokenRefresher = _asyncTokenRefresher ?? (cancellationToken => new ValueTask<string>(TokenRefresher(cancellationToken)));
             set => _asyncTokenRefresher = value;
         }
+
+        /// <summary>The proactive refreshing interval.</summary>
+        internal TimeSpan RefreshOffsetTime { get; }
+        /// <summary>Determines whether the token should be proactively renewed prior to expiry or renew on demand./// </summary>
+        internal bool RefreshProactively { get; }
+        /// <summary>The function that provides the token acquired from CommunicationIdentityClient. </summary>
+        internal Func<CancellationToken, string> TokenRefresher { get; }
+
```

Comment:
fix the linting issues - multiple blank lines
- https://dev.azure.com/azure-sdk/public/_build/results?buildId=1173607&view=logs&j=0eba20ab-c403-5626-cf18-2b71df146c5d&t=58a8e296-4048-58a7-6cbc-db9ae2b33985

Reply:
always try to format the documents by pressing `CTRL+E, D`